README.md

```diff
@@ -53,6 +53,46 @@ without SSL there are limited options for ensuring the privacy of the
 plaintext transport. See the section on [Platform authentication
 support](#platform-winrm-authentication-support).
 
+If your cloud provides you with an rdp ssl fingerprint, you can
```

Comment:
oh I just noticed #346. Would you mind replacing this documentation with the snippet from that PR? closing that one so its all in one pr?